### PR TITLE
Add `order` as module property and use it when is present before using name for ordering modules

### PR DESCRIPTION
**Pull request by jcabrerizo**
Allows to set up an order for listing the modules in the home page and the header menu. Also it renames the `App inspector` to just `Inspector` as it Apache Brooklyn can manage more than applications.

### CODE REVIEW

File: ui-modules/utils/utils/general.js [hunk 1]
```diff
@@ -54,10 +54,23 @@ export function isNonEmpty(object) {
...
+    if(moduleA.order && moduleB.order){
+        if (moduleA.order > moduleB.order) {
+            return 1;
+        }
+        if(moduleA.order < moduleB.order) {
+            return -1;
+        }
+    }
+    // If no order implemented or is the same, order by name
+    return moduleA.name > moduleB.name ? 1: -1;
```
Comment (grkvlt) [inline ui-modules/utils/utils/general.js:67]: is something like `moduleA.name.localeCompare(moduleB.name)` better?

File: ui-modules/utils/utils/general.js [hunk 2]
```diff
@@ -54,10 +54,23 @@ export function isNonEmpty(object) {
     return true; 
 }
 
+export function uiModuleComparator(moduleA, moduleB) {
+    if(moduleA.order && moduleB.order){
+        if (moduleA.order > moduleB.order) {
```
Comment (grkvlt) [inline ui-modules/utils/utils/general.js:59]: another simplification maybe `if (moduleA.order != moduleB.order) return moduleA.order - moduleB.order`

**Review by grkvlt — APPROVED** [review]
LGTM mostly

File: ui-modules/app-inspector/package.json [hunk 3]
```diff
@@ -37,7 +37,7 @@
     "npm": ">=5.0.0"
   },
   "brooklyn": {
-    "appname": "App Inspector"
+    "appname": "Inspector"
```
Comment (grkvlt) [inline ui-modules/app-inspector/package.json:40]: Should this be in this PR?

File: ui-modules/app-inspector/src/main/resources/ui-module/config.yaml [hunk 4]
```diff
@@ -15,7 +15,8 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-name: App Inspector
+name: Inspector
```
Comment (grkvlt) [inline ui-modules/app-inspector/src/main/resources/ui-module/config.yaml:18]: Same?

**Review by grkvlt — COMMENTED** [review]
Are these meant to be in [#182178851](https://www.pivotaltracker.com/story/show/182178851)

**Comment by jcabrerizo** [comment]
Thank you for the comments @grkvlt. I merge the code addressing

**Pull request merged**